docs/05-openapi/02-learning-goals.adoc

```diff
@@ -1,21 +1,34 @@
 === {learning-goals}
 
-
 // tag::DE[]
 [[LZ-5-1]]
-==== LZ 5-1: Dies ist das erste Lernziel in Kapitel 5, lorem ipsum sit dolor
-
-Hier wird beschrieben, was Teilnehmer:innen in diesem Lernziel lernen sollen. Das kann in Prosa-Text
-in ganzen Sätzen oder in Aufzählungen mit Unterpunkten erfolgen. Dabei kann auch unterschieden werden,
-wie wichtig einzelne Aspekte des Lernziels sind. Es kann hier bereits auf Literatur verwiesen werden.
+==== LZ 5-1: Wichtigkeit von API Beschreibungen kennen
 
-* Erstes Teilziel
-* Zweites Unterthema
-* Dritter Aspekt
+Teilnehmer:innen verstehen den Nutzen von API-Beschreibungen und kennen die Zielgruppen. Das Problem des "API Drift" ist bekannt. Sie können das Zusammenspiel von Design, Implementierung, Versionierung, Beschreibung, Versionierung der Beschreibung einordnen und wie diese Dinge im Idealfall sowie in der Realität praktiziert werden.
```

Comment:
API-Beschreibung besser ohne Bindestrich?

Reply:
Gleiches Thema wie bei "API Design" vs "API-Design". Ich glaube, in "API-Beschreibung" muss ein Bindestrich hin, weil "Beschreibung" ein deutsches Wort ist. Und bei "API Drift" muss keiner hin, weil "Drift" ein englisches Wort ist. Aber 100% sicher bin ich mir nicht.

Reply:
Mein Gefühl für Deutsch ist etwas angeschlagen, wahrscheinlich von englischer Grammatik. Ich finde die deutschen Bindestrich-Exzesse wenig hilfreich, aber der Duden mag da anderes denken.